src/content/docs/mobile-monitoring/new-relic-mobile-unreal-engine/monitor-your-unreal-engine-application.mdx

```diff
@@ -0,0 +1,233 @@
+
+* <DoNotTranslate>**Capture C++ errors:**</DoNotTranslate> Identify and fix problems quickly.
+* <DoNotTranslate>**Create custom events and metrics:**</DoNotTranslate> Understand how your users interact with your app.
+
+<img
+  title="Unreal Engine summary view in the UI"
+  alt="Summary view of a Unreal Engine app in New Relic"
+  src={mobileHybridSummary}
+  style={{ maxWidth: '70%' }}
+/>
+
+<figcaption>
+  <DoNotTranslate>**[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Mobile > (select an app) > Summary**</DoNotTranslate>: View Unreal Engine data, and errors, and monitor how your app is performing over time.
+</figcaption>
+
+
+## (Recommended) Guided installation [#guided-install]
+
+To install the Unreal Engine Plugin, follow our [guided install](https://onenr.io/0gR70k5oZjo), located directly in the UI.
+
+<img
+  title="Guided install for monitoring Unreal Engine mobile apps"
+  alt="Screenshot of the guided install UI for Unreal Engine"
+  src={unrealGuidedInstall}
+  style={{ maxWidth: '70%' }}
+/>
+
+## Manual installation [#manual-install]
+
+If you need to install the plugin manually, follow these steps:
+
+<Steps>
+<Step>
+### Review the requirements [#requirements]
+Before you install the Unreal Engine Plugin, make sure your Unreal Engine app meets these version requirements:
+* For Android-native apps:
+    * Android API 24 or higher
+    * See [Android-native requirements](/docs/mobile-monitoring/new-relic-mobile-android/get-started/new-relic-android-compatibility-requirements)
+* For iOS-native apps:
+    * See [iOS-native requirements](/docs/mobile-monitoring/new-relic-mobile-ios/get-started/new-relic-ios-compatibility-requirements)
+</Step>
+<Step>
+### Add the Unreal Engine Plugin to your project [#add-agent]
+
+In your game’s root directory, create a folder named Plugins Copy the NewRelic folder [https://github.com/ndesai-newrelic/newrelic-unreal-plugin/tree/master/Plugins/NewRelic]From Plugins into it. You should have something like
+
+```
+  MyProject
+  ├── Plugins
+          └── NewRelic
+              └── NewRelic.uplugin
+```
+
+</Step>
+<Step>
+Click “Restart Now” to continue configuration.
+</Step>
+<Step>
+Add NewRelic to the list of dependencies in your game module’s .Build.cs file in order to allow interfacing with NewRelic from C++ code:
+
+**PrivateDependencyModuleNames.Add("NewRelic");**
+</Step>
+<Step>
+### Add application token(s) [#app-token]
+
+To enable New Relic to authenticate your Unreal Engine app's data, follow these steps:
+
+Configure your API key through Unreal Editor’s Project Settings window, under Plugins → NewRelic.
+
+<img
+  title="Unreal Engine AppToken Setup"
+  alt="Add App Token in Unreal Engine Apps"
+  src={unrealAgentAppTokenSetup}
+  style={{ maxWidth: '70%' }}
+/>
+
+Or in Config/DefaultEngine.ini:
+[/Script/NewRelic.NewRelicSDKSettings]
+newrelicAppToken=application-token
+newrelicAppTokenIOS=ios-application-token
```

Comment:
```suggestion
In your `Config/DefaultEngine.ini` file:

```
[/Script/NewRelic.NewRelicSDKSettings]
newrelicAppToken=YOUR_APP_TOKEN
newrelicAppTokenIOS=YOUR_IOS_APP_TOKEN
```

```

@ndesai-newrelic Shouldn't newrelicAppToken=application-token be newrelicAndroidAppToken=application-token ?